element_moseq/moseq_train.py

```diff
@@ -420,18 +437,20 @@ class PreFitTask(dj.Manual):
     Attributes:
         PCAFit (foreign key)                : `PCAFit` task.
         pre_latent_dim (int)                : Latent dimension to use for the model pre-fitting.
-        pre_kappa (float)                   : Kappa value to use for the model pre-fitting.
+        pre_kappa (int)                     : Kappa value to use for the model pre-fitting.
         pre_num_iterations (int)            : Number of Gibbs sampling iterations to run in the model pre-fitting.
         pre_fit_desc(varchar)               : User-defined description of the pre-fitting task.
     """
 
     definition = """
-    -> PCAFit                                       # `PCAFit` Key
-    pre_latent_dim               : int              # Latent dimension to use for the model pre-fitting
-    pre_kappa                    : float            # Kappa value to use for the model pre-fitting
-    pre_num_iterations           : int              # Number of Gibbs sampling iterations to run in the model pre-fitting
+    -> PCAFit                                           # `PCAFit` Key
+    pre_latent_dim               : int                  # Latent dimension to use for the model pre-fitting
+    pre_kappa                    : int                  # Kappa value to use for the model pre-fitting
+    pre_num_iterations           : int                  # Number of Gibbs sampling iterations to run in the model pre-fitting

```

Comment:
It's not a good idea to add so many primary key attributes in one table. You can add instead a single primary key attribute to enumerate the prefits:
```python
class PreFitTask(dj.Lookup):
    definition = """
         -> PCAFit 
         prefit : smallint unsigned   # pre-fit number  
         ---
         pre_latent_dim               : int                  # Latent dimension to use for the model pre-fitting
         pre_kappa                    : int                     # Kappa value to use for the model pre-fitting
         pre_num_iterations           : int              # Number of Gibbs sampling iterations to run in the model pre-fitting
         pre_fit_desc=''              : varchar(1000)    # User-defined description of the pre-fitting task
         model_name                   : varchar(100)         # Name of the model to be loaded if `task_mode='load'`
         task_mode='load'             :enum('trigger','load')# 'load': load computed analysis results, 'trigger': trigger computation
         pre_fit_desc=''              : varchar(1000)        # User-defined description of the pre-fitting task
    """
```
And then reference it from `PrefitTask`

class Prefit